code/headers/st7735_inverted_color_buffered.hpp

```diff
@@ -0,0 +1,121 @@
+#pragma once
+
+#include <hwlib.hpp>
+#include <st7735_inverted_color.hpp>
+#include <type_traits>
+
+namespace r2d2::display {
+
+    /**
+     * Class st7735_inverted_color_buffered is an interface for the st7735 chip
+     * where the display has its colors inverted.
+     *
+     * Implements hwlib::window to easily use text and drawing functions that
+     * are already implemented. Extends from
+     * r2d2::display::st7735_inverted_color_c
+     *
+     * The template paramters are required for the parent class
+     */
+    template <std::size_t CursorCount, class DisplayScreen>
+    class st7735_inverted_color_buffered_c
+        : public st7735_inverted_color_c<CursorCount, DisplayScreen> {
+    protected:
+        uint16_t buffer[DisplayScreen::width * DisplayScreen::height] = {};
+
+    public:
+        /**
+         * @brief Construct a new st7735_unbuffered_c object
+         *
+         * @param bus
+         * @param cs
+         * @param dc
+         * @param reset
+         */
+        st7735_inverted_color_buffered_c(hwlib::spi_bus &bus,
+                                         hwlib::pin_out &cs, hwlib::pin_out &dc,
+                                         hwlib::pin_out &reset)
+            : st7735_inverted_color_c<CursorCount, DisplayScreen>(bus, cs, dc,
+                                                                  reset) {
+        }
+
+        /**
+         * @brief Directly write a pixel to the screen
+         *
+         * @param x
+         * @param y
+         * @param data
+         */
+        void set_pixel(uint16_t x, uint16_t y, const uint16_t data) override {
+
+            const uint16_t t =
+                __REV16(data); // make a copy and reverse byte order
```

Comment:
Same comment as the other file for this display.

Reply:
should allready be fixed in the latest version